strongbox-web-core/src/test/resources/etc/conf/strongbox-security-users.yaml

```diff
@@ -0,0 +1,19 @@
+users:
+  user:
+    - username: admin
+      password: $2a$10$WqtVx7Iio0cndyR1lEaKW.SWhUYmF/zHHG5hkAXvH5hUmklM7QfMO
+      roles:
+        - ADMIN
```

Comment:
I believe the formatting of this YAML file is incorrect. The `roles` should be like this:
```
      roles:
      - UI_MANAGER
      - ARTIFACTS_MANAGER
```

Reply:
It's perfectly valid, see https://symfony.com/doc/current/components/yaml/yaml_format.html

> YAML uses indentation with one or more spaces to describe nested collections:

Also, see our docs https://strongbox.github.io/user-guide/configuration-files/strongbox-security-users-yaml.html

and our source code https://github.com/strongbox/strongbox/blob/master/strongbox-security/strongbox-user-management/src/main/resources/etc/conf/strongbox-security-users.yaml